x-pack/plugins/maps/common/descriptor_types/layer_descriptor_types.ts

```diff
@@ -27,6 +27,7 @@ export type LayerDescriptor = {
   __isPreviewLayer?: boolean;
   __errorMessage?: string;
   __trackedLayerDescriptor?: LayerDescriptor;
+  __areTilesLoaded?: boolean;
```

Comment:
Putting in the abstract layer-descriptor because of the existing inheritance structure. It's the only common ancestor of tiled_vector_layer, vector_tile_layer/tile_layer. Not sure if this is worth a heavier refactor